internal/core/platform.rs

```diff
@@ -108,6 +108,16 @@ pub trait Platform {
         core::time::Duration::from_millis(500)
     }
 
+    /// Returns the current rate at which the text cursor should flash or blink.
+    ///
+    /// This is the length of the entire visible-hidden-visible cycle, so for a duration of 1000ms
+    /// it is visible for 500ms then hidden for 500ms, then visible again.
+    ///
+    /// If this value is `Duration::ZERO` then the cycle is disabled.
```

Comment:
An alternative would be to return an Option with none means disabled. But i think this is also fine like that.